04-Multi-Band-Rasters-In-R.Rmd

```diff
@@ -35,36 +38,44 @@ This lesson will walk through how to work with multi-band raster data in R.
 After completing this activity, you will:
 
 * Know how to identify a single vs. a multi-band raster file.
-* Be able to import multi-band rasters into `R` using the `raster` library.
-* Be able to plot multi-band color image rasters in `R` using `plotRGB`.
-* Understand what a `nodata` value is in rasters.
+* Be able to import multi-band rasters into `R` using the `raster` package.
+* Be able to plot multi-band color image rasters in `R` using `plotRGB()`.
+* Understand what a NoData value is in a raster.
```

Comment:
Again this is the difference between the concept of a No Data Value and the actual 'NoDataValue' slot in a GeoTIFF.  Here I'd leave it not in the code format.